impeller/compiler/reflector.cc

```diff
@@ -767,6 +773,75 @@ std::vector<StructMember> Reflector::ReadStructMembers(
       continue;
     }
 
+    // Tightly packed half Point (vec2).
+    if (member.basetype == spirv_cross::SPIRType::BaseType::Half &&  //
+        member.width == sizeof(float) * 4 &&                         //
+        member.columns == 1 &&                                       //
+        member.vecsize == 2                                          //
+    ) {
+      uint32_t stride =
+          GetArrayStride<sizeof(HalfVector2)>(struct_type, member, i);
+      uint32_t element_padding = stride - sizeof(HalfVector2);
+      result.emplace_back(StructMember{
+          "HalfVector2",                         // type
```

Comment:
Argument comments are of the format `/*arg_name=*/`: https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments

Reply:
This is following the style of the rest of the file.

Reply:
...alos this is a struct, not a function call.

Reply:
Not a function call, but with C++20, you can use aggregate initialization and do this without comments: `{.type = "HalfVector2"}`.